README.md

```diff
@@ -1,83 +1,64 @@
-# msvc-code-analysis-action
+# Microsoft C++ Code Analysis Action
 
-This action enables code analysis to run while building a project with the Microsoft Visual C++ Compiler. The analysis
-will produce SARIF results that can be uploaded to the GitHub Code Scanning Alerts experience.
+This actions run code analysis for any CMake project built with the Microsoft Visual C++ Compiler. The analysis
+will produce SARIF results that can be uploaded to the GitHub Code Scanning Alerts experience and/or included as
+an artifact to view locally in the Sarif Viewer VSCode Extension.
 
 ## Usage
 
 ### Pre-requisites
 
 Include a workflow `.yml` file using an [example](#example) below as a template. Run the `msvc-code-analysis-action`
```

Comment:
"file using [example](#example) below as a template" -- The word 'an' should be removed.